spec/system/courses/students_spec.rb

```diff
@@ -55,6 +69,10 @@
       expect(page).to have_text(startup.founders.first.name)
     end
 
+    # Check the last seen for the first student
+    expect(page).to have_text(first_student.name)
+    expect(page).to have_text('Last seen: 3 days ago')
```

Comment:
To improve the precision of the test, it's good to also ensure that this information is present in the correct element in the page. 

Like in few other places in the same spec.

```ruby
 within("div[aria-label='.......']") do
     expect(page).to have_text('Last seen: 3 days ago')
 end
  ```
  Fill the selector information after checking the logic for the element holding the student name and tags.

  This kind of double-checks the expected data is in the right place and not somewhere in the page.